components/carousel/carouselMarkers.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function CarouselMarkers({ images, current, onClick }) {
+  return (
+    <div className="ui-carousel-markers">
+      {images.map((src, i) => (i === current ? <i key={i} /> : <button data-index={i} key={i} onClick={onClick} />))}
```

Comment:
`<i key={i} />` this is not sematically right
https://www.w3.org/International/questions/qa-b-and-i-tags they it should represent *italic* content